src/cli/pack_command.zig

```diff
@@ -1359,7 +1359,9 @@ pub const PackCommand = struct {
                         var files_array = _files_array;
                         while (files_array.next()) |files_entry| {
                             if (files_entry.asString(ctx.allocator)) |file_entry_str| {
-                                const parsed = try Pattern.fromUTF8(ctx.allocator, file_entry_str) orelse continue;
+                                var path_buf: PathBuffer = undefined;
+                                const normalized = bun.path.normalizeBuf(file_entry_str, &path_buf, .posix);
+                                const parsed = try Pattern.fromUTF8(ctx.allocator, try ctx.allocator.dupe(u8, normalized)) orelse continue;
```

Comment:
```suggestion
                                const parsed = try Pattern.fromUTF8(ctx.allocator, normalized) orelse continue;
```
We don't need to clone this memory because `ctx.allocator` will clone it when it's converted to utf32 in `fromUTF8`